src/index.ts

```diff
@@ -1 +1,6 @@
-console.log('Hello, World!');
+import { GameField } from './gamefield';
+
+const gameField = new GameField(' ', 10, 10)
```

Comment:
use constants or proccess.argv to set size

Reply:
"Magic numbers"

Reply:
Done